sway/commands/opacity.c

```diff
@@ -4,11 +4,23 @@
 #include "sway/tree/view.h"
 #include "log.h"
 
-static bool parse_opacity(const char *opacity, float *val) {
+static bool adjust_opacity(const char *opacity, float *alpha) {
 	char *err;
-	*val = strtof(opacity, &err);
-	if (*val < 0 || *val > 1 || *err) {
-		return false;
+	float val = 0.0f;
```

Comment:
Is this temp assignment really necessary? should be able to combine this and actual assignment below.

Reply:
Good point, I'll fix this.